shared/menubar/index.js

```diff
@@ -187,8 +187,8 @@ class Menubar extends Component {
 
 export default connect(
   state => ({
-    username: state.config && state.config.status && state.config.status.user && state.config.status.user.username,
-    loggedIn: state.config && state.config.status && state.config.status.loggedIn,
+    username: state.config.username,
```

Comment:
> Seems that state.config isn't defined at the time that it opens.

Correct, the RemoteCompnent abstraction has a prop that will wait for the store to be loaded before rendering the component: (https://github.com/keybase/client/blob/master/desktop/renderer/remote-component-loader.js#L100)


But menubar doesn't use that (that's fine because it doesn't quite make sense, RemoteComponent is when you want to make a popup/remote-window from existing window. This is just a remote-window spawned from the main node process).

The higher level fix would be to do something similar to check if the store is loaded before rendering Menubar.
here: https://github.com/keybase/client/blob/master/desktop/renderer/launcher.js#L29

lmk if it doesn't quite make sense and I'll elaborate :)